butte/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "butte"
+version = "0.1.0"
+authors = ["Phillip Cloud <cpcloud@gmail.com>"]
+description = "A Rust implementation of flatbuffers"
+license = "MIT"
+repository = "https://github.com/butte-rs/butte"
+readme = "README.md"
+edition = "2018"
+keywords = [
+    "flatbuffers",
+    "flatbuffer",
+    "codegen",
+    "data",
+    "performance",
+    "parsing",
+    "encoding",
+    "encode",
+    "decoding",
+    "decode",
+    "serialization",
+    "serialize",
+    "deserialization",
+    "deserialize",
+    "rpc",
+    "grpc",
+    "service",
+]
+categories = ["data-structures", "encoding", "parser-implementations"]
+
+[dependencies]
+anyhow = "1.0.19"
+derive_more = "0.15.0"
+heck = "0.3.1"
+hexf-parse = "0.1.0"
+nom = { version = "5.0.1", features = ["regexp"] }
+proc-macro2 = "1.0.6"
+quote = "1.0.2"
+syn = "1.0.7"
+typed-builder = "0.3.0"
+structopt = "0.3.3"
```

Comment:
Clap has merged structopt's code in and is afaik the recommended crate again to bind command line arguments to a struct 🙂

Reply:
Oh, sweet.

Reply:
It looks like this isn't ready yet: https://github.com/clap-rs/clap_derive

Reply:
Oh yeah, the `clap` page itself doesn't have this huge warning and suggests:
```toml
[dependencies]
clap = "~3.0.0-beta.1"
```
(where the `derive` feature is enabled by default).

But yeah, let's stick with the stable version for now